modules/manage/partials/mountable-topics.adoc

```diff
@@ -0,0 +1,279 @@
+
+== Prerequisites
+
+[NOTE]
+====
+include::shared:partial$enterprise-license.adoc[]
+====
+
+. xref:get-started:rpk-install.adoc[Install `rpk`], or ensure that you have access to the Admin API.
+. Enable xref:manage:tiered-storage.adoc[Tiered Storage] for specific topics, or for the entire cluster (all topics).
+
+== Unmount a topic from a cluster to object storage
+
+When you unmount a topic, all incoming writes to the topic are blocked as Redpanda unmounts the topic from the cluster to object storage. Producers and consumers of the topic receive a message in the protocol replies indicating that the topic is no longer available:
+
+- Produce requests receive an `invalid_topic_exception` or `resource_is_being_migrated` response from the broker.
+- Consume requests receive an `invalid_topic_exception` response from the broker.
+
+An unmounted topic in object storage is detached from all clusters. The original cluster releases ownership of the topic.
+
+NOTE: The unmounted topic is deleted in the source cluster, but can be mounted back again from object storage.
+
+[tabs]
+======
+rpk::
++
+--
+In your cluster, run this command to unmount a topic to object storage:
+
+```
+rpk cluster storage unmount <namespace>/<topic-name>
+```
+--
+Admin API::
++
+--
+To unmount topics from a cluster using the Admin API, make a POST request to the `/v1/topics/unmount` endpoint.  Specify the names of the desired topics in the request body:
+
+```
+curl -X POST http://localhost:9644/v1/topics/unmount -d {
+  "topics": [
+    {
+      "topic": "<topic-1-name>"
+    }, 
+    {
+      "topic": "<topic-2-name>"
+    }, 
+    {
+      "topic": "<topic-3-name>"
+    }
+  ]
+}
+```
+
+You may optionally include the topic namespace (`ns`). Only `kafka` is supported.
+--
+======
+
+You can use the ID returned by the command to <<monitor-progress,monitor the progress>> of the unmount operation using `rpk` or the Admin API.
+
+== Mount a topic to a cluster
+
+[tabs]
+======
+rpk::
++
+--
+. In your target cluster, run this command to list the topics that are available to mount from object storage:
++
+```
+rpk cluster storage list-mountable
+```
++
+The `LOCATION` value returned in the command output is in the format `<topic-name>/<cluster-uuid>/<initial-revision>`. For example:
++
+```
+TOPIC      NAMESPACE  LOCATION
+testtopic  kafka      testtopic/67f5505a-32f3-4677-bcad-3c75a1a702a6/10
+```
++
+Redpanda assigns the number `initial-revision` to a topic upon creation. You can use the topic location instead of just the topic name to uniquely identify a topic to mount in the next step. 
+
+. Mount a topic from object storage:
++
+```
+rpk cluster storage mount <source-topic-reference>
+```
++
+Replace `<source-topic-reference>` with the name of the topic to mount, or the topic location to uniquely identify a topic object storage.
++
+You can also rename the topic as you mount it to the target cluster:
```

Comment:
@mattschumpert @nvartolomei does this mean, the alias does not change the name of the _remaining mountable topics_ of the same name in cloud storage?

Reply:
I will defer to @bashtanov but as far as I understand alias is a better term here. We don't rename any topics we just present it with a different name.

Reply:
It definitely does not change the name of the mountable topic. It does "change" the name of the [cluster-mounted] topic in a sense that it may have been different before, and was different originally. I'd write something like "specify a different topic name" instead of "change", as it won't even always change within this operation.

Example:
1. Create topic `foo`.
2. Unmount topic `foo`, so that it becomes a mountable called `foo`.
3. Mount the mountable `foo` as topic `bar`.
4. Unmount topic `bar`, it becomes a mountable called `foo` again, as underlying storage name always remains the same.
5. Remount the mountable `foo` as topic `bar`.

Using an alias in (5) does not make (4-5) change the topic name, but rather makes it keep it.

Reply:
Rephrased, checked via Slack